spec/reek/smells/attribute_spec.rb

```diff
@@ -47,26 +35,10 @@ class Klass
       ').to_not reek_of(:Attribute)
     end
 
-    it 'records attr attribute in a module' do
+    it 'records attr_writer attribute in a module' do
       expect('
         module Mod
-          attr :my_attr
-        end
-      ').to reek_of(:Attribute, name: 'my_attr')
-    end
-
-    it 'records attr attribute' do
-      expect('
-        class Klass
-          attr :my_attr
-        end
-      ').to reek_of(:Attribute, name: 'my_attr')
-    end
-
-    it 'records reader attribute' do
-      expect('
-        class Klass
-          attr_reader :my_attr
+          attr_writer :my_attr
         end
```

Comment:
It looks like `SendNode#arg_names` is a bit of a hack that assumes all arguments are symbols. You can use `SendNode#args` instead, which will give you the real arguments.